packages/main/src/DatePicker.ts

```diff
@@ -865,7 +866,7 @@ class DatePicker extends DateComponentBase implements IFormElement {
 			CalendarDateComponent.default,
```

Comment:
Do we need this import? CalendarDate has only 1 default export. If its import is replaced we can removed the next casting.

For me this could be replaced with:
```ts
From: 
import * as CalendarDateComponent from "./CalendarDate.js";
To:
import CalendarDateComponent from "./CalendarDate.js";
```

Then instead of `as ...` you can add just return type to the function `static get dependencies(): Array<typeof UI5Element> {`